frontend/integration-tests/tests/olm/crd.scenario.ts

```diff
@@ -0,0 +1,45 @@
+import { browser, $, $$, by, ExpectedConditions as until, element } from 'protractor';
+import { appHost, checkLogs, checkErrors } from '../../protractor.conf';
+
+describe('Test for CRD related pages', () => {
+
+  afterEach(() => {
+    checkLogs();
+    checkErrors();
+  });
+
+  it('Check CatalogSource instances', async() => {
+    browser.get(`${appHost}/k8s/cluster/customresourcedefinitions`);
+    await browser.wait(until.presenceOf($('.ReactVirtualized__Grid__innerScrollContainer')));
+
+    const cs = element(by.linkText('CatalogSource'));
+    expect(cs.getAttribute('href')).toContain('catalogsources.operators.coreos.com');
+    await cs.click();
+
+    await browser.wait(until.presenceOf($('.btn-dropdown__item')));
+    await $('.btn-dropdown__item').click();
+    const viewInstance = element(by.xpath('//a[contains(text(),"View Instances")]'));
+    await viewInstance.click()
+
+    await browser.wait(until.presenceOf($('.dropdown-kebab-pf')));
+    expect($$('.ReactVirtualized__Grid__innerScrollContainer .co-m-row').count()).toBeGreaterThan(0);
+  });
+
+  it('Check CatalogSource instances without OperatorGroup - certified-operators', async() => {
+    browser.get(`${appHost}/k8s/ns/openshift-marketplace/operators.coreos.com~v1alpha1~CatalogSource/certified-operators`);
```

Comment:
here we'd better use click operation instead of using URL for redirection

Reply:
@yapei Agree with you at the first thought, but when the first step is failed in somewhere before checking this page, it blocks the following checking. HDYT?